app/src/main/java/de/westnordost/streetcomplete/data/osmnotes/edits/NoteEdit.kt

```diff
@@ -31,8 +32,11 @@ data class NoteEdit(
     /** whether this edit has been uploaded already */
     override val isSynced: Boolean,
 
-    /** Whether the images attached still need activation. Already true if imagePaths is empty */
-    val imagesNeedActivation: Boolean
+    /** whether the images attached still need activation. Already true if imagePaths is empty */
+    val imagesNeedActivation: Boolean,
+
+    /** attached GPS location tracks */
+    val tracks: List<Trackpoint>,
```

Comment:
This only stores a single track, right? (One track = list of trackpoints)

Then please rename this variable from `track` to `track` and adjust calling functions, the database column name and code comments accordingly.

I think it was this missing distinction that confused me a lot when I tried to implement saving to the instance.

Reply:
Renamed everything to `track` and left the existing `trackpoints` names as is.